crates/core/liveview-native-core-jetpack/core/build.gradle.kts

```diff
@@ -13,16 +13,22 @@ subprojects {
 dependencies {
     testImplementation("junit:junit:4.13.2")
     androidTestImplementation("androidx.test.ext:junit:1.1.5")
-    androidTestImplementation("androidx.test.espresso:espresso-core:3.4.0")
+    androidTestImplementation("androidx.test.espresso:espresso-core:3.5.1")
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4")
-    implementation("net.java.dev.jna:jna:5.13.0")
+    testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:1.7.3")
+    implementation("net.java.dev.jna:jna:5.14.0")
+    coreLibraryDesugaring("com.android.tools:desugar_jdk_libs:2.0.4")
 }
 val uniffiPath = "${buildDir}/generated/source/uniffi/java"
 
 android {
     namespace = "org.phoenixframework.liveview_native_core_jetpack"
     compileSdk = 33
 
+    lint {
+         disable += "NewApi"
+    }
```

Comment:
@nelson-glauber uniffi generates stuff [using `java.lang.ref.Cleaner` which is Android API 33](https://developer.android.com/reference/java/lang/ref/Cleaner) but has a [try-catch at runtime to switch to the JNA cleaner](https://github.com/mozilla/uniffi-rs/pull/1869#issuecomment-1881912358). I don't think that this lint is correct as I think it catches all `NewApi` parts but I'm not sure how to specify just `java.lang.ref.Cleaner` and associated functions.

Reply:
Yeah... I believe there are a couple of better options to deal with this... 
Using this option, you're disabling the "NewApi" lint warning in the entire module. You could, for instance, ignore just a specific file (as described [here](https://developer.android.com/studio/write/lint#sample-lint.xml-file)).

However, I believe that a better option would be to wrap the `Cleaner` usage with a version check block.
```kotlin
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
    useJavaCleaner()
} else {
    useJNACleaner()
}
```
And declare the functions...
```kotlin
@TargetApi(Build.VERSION_CODES.TIRAMISU)
fun useJavaCleaner() {
   // use java.lang.ref.Cleaner
}

fun useJNACleaner() {
    // Use JNACleaner...
}
```

Reply:
Cool. I added some specific parts in the `lint.xml`. Thanks!